scripts/performance-helper.psm1

```diff
@@ -1241,18 +1245,16 @@ class TestRunDefinition {
         $this.AllowLoopback = $existingDef.AllowLoopback
         $this.Formats = $existingDef.Formats
         $this.RegressionThreshold = $existingDef.RegressionThreshold
+        $this.XDP = $script:XDP
     }
 
     [string]ToString() {
         $VarVal = "_$($this.VariableValue)"
         if ($this.VariableName -eq "Default") {
             $VarVal = ""
         }
-        $Platform = $this.Local.Platform
-        if ($script:Kernel -and $this.Local.Platform -eq "Windows") {
-            $Platform = 'Winkernel'
-        }
-        $RetString = "$($this.TestName)_$($Platform)_$($script:RemoteArch)_$($script:RemoteTls)_$($this.VariableName)$VarVal"
+
+        $RetString = "$($this.TestName)_$($this.ToTestPlatformString())_$($this.VariableName)$VarVal"
```

Comment:
@thhous-msft can you please review this?

Reply:
I did. Its fine. Was waiting to approve until everything passed.

Reply:
windows user openssl perf test failed on the tcp ones. I don't think it's related to this PR.

Running Local: C:\RunningTests\secnetperf.exe Args: -test:Throughput -target:quic-server -bind:192.168.0.2:4434 -ip:4 -uni:1 -download:2000000000 -tcp:1
Running Locally: C:\RunningTests\secnetperf.exe -watchdog:115000 "--extraOutputFile:C:\RunningTests\ExtraRunFile.txt" -test:Throughput -target:quic-server -bind:192.168.0.2:4434 -ip:4 -uni:1 -download:2000000000 -tcp:1
Started!

Started!

App Main returning status 0